AlertSender.ps1

```diff
@@ -352,37 +352,64 @@ try {
 			# Create variable for service name in TitleCase format.
 			$textInfo = (Get-Culture).TextInfo
 			$serviceName = $textInfo.ToTitleCase($service.Name)
+			If ($service.Value.webhook -ne $null) {
+				If ($service.Value.webhook.StartsWith('https')) {
+					Write-LogMessage -Tag 'INFO' -Message "Sending notification to $($serviceName)."
+					$logId_service = $vbrSessionLogger.AddLog("[VeeamNotify] Sending notification to $($serviceName)...")
+
+					# Add user information for mention if relevant.
+					Write-LogMessage -Tag 'DEBUG' -Message 'Determining if user should be mentioned.'
+					If ($mention) {
+						Write-LogMessage -Tag 'DEBUG' -Message 'Getting user ID for mention.'
+						$payloadParams.UserId = $service.Value.user_id
+
+						# Set username if exists
+						If ($service.Value.user_name -and $service.Value.user_name -ne 'Your Name') {
+							Write-LogMessage -Tag 'DEBUG' -Message 'Setting user name for mention.'
+							$payloadParams.UserName = $service.Value.user_name
+						}
+					}
 
-			If ($service.Value.webhook.StartsWith('https')) {
-				Write-LogMessage -Tag 'INFO' -Message "Sending notification to $($serviceName)."
-				$logId_service = $vbrSessionLogger.AddLog("[VeeamNotify] Sending notification to $($serviceName)...")
+					# Get URI from webhook value
+					$uri = $service.Value.webhook
 
```

Comment:
What's the reason for these changes?

Reply:
I selected the wrong line numbers for this comment but I'm referring to everything except the first and last change here:
![image](https://github.com/tigattack/VeeamNotify/assets/10629864/6ce088ce-7557-4842-aed7-0a8b6545bc32)

Reply:
Are you referring to the $Uri being moved or reassigned? Sorry, not sure what you mean.

Reply:
Sorry, I forget exactly what I was querying now, but I'd guess that's what I meant. We can leave this since it's not a big deal, not sure what I had in mind at the time.